src/main/java/warmup/Arizona.java

```diff
@@ -0,0 +1,135 @@
+package main.java.warmup;
```

Comment:
main.java - should not be in package name (it is a dir where you place your code, let's say it is a root dir for sources)